templates/impala/search.html

```diff
@@ -3,7 +3,10 @@
   {% set search_placeholder = 'all' %}
 {% endif %}
 {% set search_form = SimpleSearchForm(request, search_cat) %}
+
 <form id="search" action="{{ search_url }}">
+    <div role="status" aria-live="polite" id="announce-suggestions"></div>
+    <label for="search-q" hidden>type to search for add-ons</label>
```

Comment:
Running a real one is likely going to be a better approach than trying to use a simulator if you want to try it out. Here's a couple of articles written from the perspective of testing (both for windows) with JAWs http://webaim.org/articles/jaws/ and another for using NVDA: http://webaim.org/articles/nvda/ . If you're on a mac then you can try with voiceover. http://webaim.org/articles/voiceover/